.github/workflows/update-sdk-docs.yaml

```diff
@@ -0,0 +1,69 @@
+name: Update SDK Docs
+
+on: 
```

Comment:
we can also add cron schedule [^1] to run this daily as described by the issue #307 

[^1]: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule